src/components/transactions/modals/ClaimWRBTCModal.vue

```diff
@@ -159,12 +169,26 @@ export default {
           break
         }
         case this.claimTypes.CONVERT_TO_RBTC: {
-          const weiAmount = new BigNumber(this.receiveAmount).shiftedBy(-8).toString()
+          this.processing = true
+          
+          const weiAmount = new BigNumber(this.receiveAmount).shiftedBy(8).toFixed(0)
```

Comment:
we should take the decimals from the [src/constants/tokens] (https://github.com/rsksmart/tokenbridge-ui/blob/main/src/constants/tokens/testNetKovan.js#L114)
As WBTC has 8 decimals but Binance BTC has 18

Reply:
ok added an item to the backlog https://rsklabs.atlassian.net/browse/TBRE-516